src/Selenium2Driver.php

```diff
@@ -541,7 +541,7 @@ public function getText($xpath)
     {
         $node = $this->findElement($xpath);
         $text = $node->text();
-        $text = (string) str_replace(array("\r", "\r\n", "\n"), ' ', $text);
```

Comment:
Won't swapping 2nd and 1st array element solve the issue as well?

Reply:
`\r\n` to 2 spaces yes, but other whitespaces normalization not (like if some browser keeps tabs etc.). Plus this PR also trim the ends.